azurerm_api_management_products/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "environment" {
+  description = "The nick name identifying the type of environment (i.e. test, staging, production)"
```

Comment:
. at the end

Reply:
Still needs to be fixed

Reply:
has been copied from other modules, need a full replace